content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -106,11 +129,24 @@ Product-specific dynamic text values are as follows:
       :alt: PDF quote being built using common dynamic placeholders.
 
 Once the PDF file(s) are complete, save them to the computer's hard drive, and proceed to upload
-them to Odoo via :menuselection:`Sales app --> Configuration --> Settings --> PDF Quote builder`.
+them to Odoo via :menuselection:`Sales app --> Configuration --> Headers/Footers`.
 
-Upload the created PDF in the :guilabel:`Header pages` or :guilabel:`Footer pages` field.
+.. example::
+   When uploading a **header** PDF containing the form field :guilabel:`invoice_partner_country`,
+   configuring the :guilabel:`path` of the :guilabel:`Form Field Name` to
+   :guilabel:`partner_invoice_id.country_id.name` fills the form with the invoice partner country's
+   name when the PDF is built.
 
-Once the upload(s) are complete, click :guilabel:`Save`.
+.. example::
+   When uploading a **product** PDF containing the form field :guilabel:`invoice_partner_country`,
+   configuring the :guilabel:`path` of the :guilabel:`Form Field Name` to
+   :guilabel:`order_id.partner_invoice_id.country_id.name` fills the form with the invoice partner
+   country's name when the PDF is built.
+
+.. example::
+   When uploading any PDF containing the form field :guilabel:`custom_note`, leaving the
+   :guilabel:`path` empty allows the seller to write down any note where that form field is in that
+   document and shown when the PDF is built.
```

Comment:
A totally optional suggestion here is using a single example attribution, with the content of each example block as bullet points.